Networking/Networking/Model/Stats/OrderStatsV4Totals.swift

```diff
@@ -0,0 +1,102 @@
+/// Represents the data associated with order stats over a specific period.
+/// v4
+public struct OrderStatsV4Totals: Decodable {
+    public let orders: Int
+    public let itemsSold: Int
+    public let grossRevenue: Decimal
+    public let couponDiscount: Decimal
+    public let coupons: Int
```

Comment:
nit: for plural names like `orders`, `itemsSold`, `coupons`, my first thought is that they are an array of orders/items/coupons. I felt a name like `numberOf*` or `*Count` would make it sound exactly what is it. Especially there are `refunds` below, which is the amount of refund not the number of refunds

Reply:
Good point. I'll go with total* which matches the nomenclature in the previous stats, does the sound better?

Reply:
oops sorry for responding late! yep `total*` sounds great 👍